lib/elixir/lib/uri.ex

```diff
@@ -279,14 +280,14 @@ defmodule URI do
       "a str%69ng"
 
   """
-  @spec encode(binary, (byte -> boolean)) :: binary
+  @spec encode(binary, (char -> as_boolean(term))) :: binary
```

Comment:
```suggestion
  @spec encode(binary, (byte -> as_boolean(term))) :: binary
```

Below we are taking bytes out of the binary (although we incorrectly named the var `char`).

Reply:
I am renaming everything `byte` then.